### DISCUSSION TITLE
How to group VLANs when they are not tied to a location

### DISCUSSION THREAD

**Discussion by moffitj** [netbox-community/netbox#14324] [Q&A]
It seems like VLAN groups can only be scoped to a physical location, or a group of physical locations. I'd like to limit the VLAN group to the purpose of the device instead, so that Network A and Network B can only be assigned VLANs from their own group, even though they are in the same physical location.

I have an OOB network that spans multiple sites. I would like to set up a VLAN group so that only a specific range of VLANs can be assigned to these devices. I have other networks A through G that I want to do the same thing for. All of these networks are physically in the same sites.

The only way I can see to do this is to create an individual site for each network per site. But then that breaks a lot of other relations. For example a Rack can only be in one site.

Main Campus - OOB
Main Campus - Network A
Main Campus - Network B

**Comment by Chiniquy** [comment]
We solved this (or created a work-around I guess) by using Scope Type = Cluster, and then making a Cluster that includes those specific devices, but I imagine that this would not work for all environments. I would agree that there is some unexplored space here for VLAN Group Scopes.

**Reply by moffitj** [reply]
Could you elaborate at all? I just tried this and it didn't work.

Create a virt cluster group and virt clusters
Created a VLAN group and scoped it to a virt cluster
Created a device in the virt cluster and tried assiging a VLAN to one of the interfaces.

It still gave me the option for all of the vlan groups, but despite that I wasn't able to specify a single tag.
